cfme/common/__init__.py

```diff
@@ -274,7 +274,7 @@ def _assign_tag_action(self, view, tag):
         """
         if not tag:
             tag = self._set_random_tag(view)
-        category_name = tag.category.display_name
+        category_name = tag.category
```

Comment:
This is not correct. category_name should be a string, not an object. 

https://github.com/ManageIQ/integration_tests/blob/master/cfme/configure/configuration/region_settings.py#L75

Reply:
@jawatts For some reason ```tag.category``` does not contain the attr ```display_name```
when I investigated this I found that ```tag. category``` is the category name.

Reply:
@shalomnaim1 That is because you need to instantiate the category object. In the smart management test, you were passing it as a string.

Reply:
@jawatts So what it the right way to use this object?
could you please show me an example?

Reply:
@shalomnaim1 Here is an example: https://github.com/ManageIQ/integration_tests/blob/master/cfme/tests/cloud_infra_common/test_vm_instance_analysis.py#L768